userguide/tutorials/development.adoc

```diff
@@ -1,129 +1,530 @@
 
-Some of the Maven plugins used may not be available to m2e, however, and the projects will not import completely. To work around this issue, the Eclipse `Preferences > Maven > Lifecycle Mappings` configuration can be updated to include mappings like the following:
+==== Cloning the Repository
 
-[source,bash]
-----
-<?xml version="1.0" encoding="UTF-8"?>
-<lifecycleMappingMetadata>
- <pluginExecutions>
-   <pluginExecution>
-     <pluginExecutionFilter>
-       <groupId>org.apache.maven.plugins</groupId>
-       <artifactId>maven-antrun-plugin</artifactId>
-       <versionRange>1.4</versionRange>
-       <goals>
-         <goal>run</goal>
-       </goals>
-     </pluginExecutionFilter>
-     <action>
-       <execute>
-         <runOnConfiguration>true</runOnConfiguration>
-         <runOnIncremental>true</runOnIncremental>
-       </execute>
-     </action>
-   </pluginExecution>
-   <pluginExecution>
-     <pluginExecutionFilter>
-       <groupId>org.codehaus.mojo</groupId>
-       <artifactId>templating-maven-plugin</artifactId>
-       <versionRange>1.0-alpha-3</versionRange>
-       <goals>
-         <goal>filter-sources</goal>
-       </goals>
-     </pluginExecutionFilter>
-     <action>
-       <execute>
-         <runOnConfiguration>true</runOnConfiguration>
-         <runOnIncremental>true</runOnIncremental>
-       </execute>
-     </action>
-   </pluginExecution>
-   <pluginExecution>
-     <pluginExecutionFilter>
-       <groupId>org.codehaus.mojo</groupId>
-       <artifactId>build-helper-maven-plugin</artifactId>
-       <versionRange>[1.0,)</versionRange>
-       <goals>
-         <goal>add-source</goal>
-         <goal>add-test-source</goal>
-         <goal>add-resource</goal>
-         <goal>add-test-resource</goal>
-         <goal>maven-version</goal>
-         <goal>parse-version</goal>
-       </goals>
-     </pluginExecutionFilter>
-     <action>
-       <execute>
-         <runOnConfiguration>true</runOnConfiguration>
-         <runOnIncremental>true</runOnIncremental>
-       </execute>
-     </action>
-   </pluginExecution>
- </pluginExecutions>
-</lifecycleMappingMetadata>
-----
+The first step is to clone the Kill Bill repository. For this, you need to follow the steps given below:
 
+. Start Eclipse
 
-=== Build
+. Go to *Git Repository view* (Click on `Window > Show View > Other > Git Repositories`)
 
-Kill Bill is a standard Maven project. Simply run the following command from the https://github.com/killbill/killbill[project] root directory:
+. Click on *Clone a Git Repository and add the clone to this view*
 
-[source,bash]
-----
-mvn clean install -DskipTests
-----
+. In the Git Repository details, enter the Kill Bill Git URL https://github.com/killbill/killbill in the URI field. Enter your Git Credentials. Click *Next*
```

Comment:
Are credentials really required? I believe one could clone the repository anonymously.

Reply:
Updated document to include steps for cloning the repository anonymously